example/ios/PrivacyInfo.xcprivacy

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Just to keep cocoapods happy since it kept complaining about this. We don't really need it since we don't publish this app to TF/public.